bricks/config.rb

```diff
@@ -127,7 +127,7 @@ def custom_config(config)
 # or, to map host folders to guest folders of the same name,
 # $shared_folders = Hash[*['/home/foo/app1', '/home/foo/app2'].map{|d| [d, d]}.flatten]
 #$shared_folders = {}
-$shared_folders = {'/var/log/amobee' => '/var/log/amobee', '/etc/amobee' => '/etc/amobee'}
+$shared_folders = {'/var/log/amobee' => '/var/log/amobee', '/etc/amobee' => '/etc/amobee',  '/var/amobee' => '/tmp/var/amobee'}
```

Comment:
I thought we might want to retain the data in there folders. Attaching to `/tmp/**` means we can lose that data anytime ?